README.md

```diff
@@ -1,13 +1,18 @@
 # IO
 
-Go packages for pheripheral I/O. It contains driver for the following chips:
+Go packages for pheripheral I/O. It contains driver for the following IC's:
 
 * SPI
     * [Microchip](spi/microchip)
         * MCP3004
         * MCP3008
         * MCP3204
         * MCP3208
+* I<sub>2</sub>C
+    * [Maximum Integrated](i2c/max)
```

Comment:
I think is it's more sophisticated to reference to online godoc instead of documents in repo.